packages/devextreme/js/__internal/ui/tree_view/m_tree_view.base.ts

```diff
@@ -801,11 +801,13 @@ const TreeViewBase = (HierarchicalCollectionWidget as any).inherit({
 
   _addContent($container, itemData) {
     const { html, url } = itemData;
-
-    if (url) {
-      $container.html(html);
-      const link = this._getLinkContainer(this._getIconContainer(itemData), this._getTextContainer(itemData), itemData);
-      $container.append(link);
+    if (this.option('adaptivityEnabled')) {
```

Comment:
[naming]
Why `adaptivityEnabled`? Let's make it more clear. Also kindly:
- add an underscore as a prefix to notice it's a private option
- set a default value in TreeView
- add an empty option change handler
```suggestion
    if (this.option('_supportItemUrl')) {
```

Reply:
[d45533d](https://github.com/DevExpress/DevExtreme/pull/27647/commits/d45533db3d0fdfffa3fcf4d8a704612c6f429a4e)